web/src/main/scala/net/wiringbits/components/widgets/EditUserForm.scala

```diff
@@ -0,0 +1,121 @@
+package net.wiringbits.components.widgets
+
+import com.alexitc.materialui.facade.materialUiCore.{components => mui, materialUiCoreStrings => muiStrings}
+import net.wiringbits.api.models.GetCurrentUser
+import net.wiringbits.forms.UpdateInfoFormData
+import net.wiringbits.ui.components.inputs.{EmailInput, NameInput}
+import net.wiringbits.webapp.utils.slinkyUtils.components.core.ErrorLabel
+import net.wiringbits.webapp.utils.slinkyUtils.components.core.widgets.{CircularLoader, Container}
+import net.wiringbits.webapp.utils.slinkyUtils.forms.StatefulFormData
+import net.wiringbits.{AppContext, AppStrings}
+import org.scalajs.dom
+import org.scalajs.macrotaskexecutor.MacrotaskExecutor.Implicits.global
+import slinky.core.annotations.react
+import slinky.core.facade.{Fragment, Hooks}
+import slinky.core.{FunctionalComponent, SyntheticEvent}
+import slinky.web.html._
+
+import scala.util.{Failure, Success}
+
+@react object EditUserForm {
+  case class Props(ctx: AppContext, jwt: String, user: GetCurrentUser.Response, onSave: () => Unit)
```

Comment:
The `jwt` isn't necessary, is it? I think we have the `User` model which includes it, I'd propagate that instead